packages/flame/lib/src/components/time_track_component.dart

```diff
@@ -0,0 +1,55 @@
+import 'dart:async';
+import 'dart:ui';
+
+import 'package:flame/components.dart';
+import 'package:flame/text.dart';
+
+class TimeTrackComponent extends TextComponent {
+  TimeTrackComponent({super.position});
+
+  static final Map<String, int> _startTimes = {};
+  static final Map<String, int> _endTimes = {};
+
+  static void clear() {
+    _startTimes.clear();
+    _endTimes.clear();
+  }
```

Comment:
Hmm, do we really want all these to be static?

Reply:
Yeah this is also part of the rough implementation. But at the same time, I thought that keeping then static would make the tracking be less impactful and make the tracking more accurate? But anyhow totally open to suggestion, at the time I just did it static because I needed something quick.

Reply:
Yeah, if it is an explicit debug component it can probably just be kept as static, since debugging is supposed to be "quick & dirty" anyways.